website/docs/components/side-nav/partials/code/how-to-use.md

```diff
@@ -406,7 +402,7 @@ When the Side Nav is paired with the [`Hds::AppHeader`](/components/app-header)
     </:body>
   </Hds::SideNav>
 </div>
-```
+``` -->
```

Comment:
Should this be removed instead of commented as, as the ones above were?

Reply:
Since it included code I thought maybe it would be better to just comment out since it would take a little more to recreate than just updating text.